src/amber/cli/recipes/file_entries.cr

```diff
@@ -45,7 +45,7 @@ module Amber::Recipes
           template = Liquid::Template.parse filename
           filename = template.render @ctx.as(Liquid::Context)
 
-          if /^(.+)\.lqd$|^(.+)\.liquid$/ =~ filename
+          if /^(.+)\.lqd$/ =~ filename || /^(.+)\.liquid$/ =~ filename
```

Comment:
This change is fine, but let's try to make the regex constants for readability and a least one spec for the change.